SECURITY_RELEASE_PROCESS.md

```diff
@@ -188,20 +188,28 @@ could be in the form of the following:
 To be eligible for the cncf-envoy-distributors-announce mailing list, your
 distribution should:
 
-1. Be an actively maintained distribution of Envoy components OR offer Envoy as a publicly
-   available service in which the product clearly states that it is built on top of Envoy. E.g.,
-   "SuperAwesomeLinuxDistro" which offers Envoy pre-built packages OR
-   "SuperAwesomeCloudProvider's Envoy as a Service (EaaS)". A cloud service that uses Envoy for a
-   product but does not publicly say they are using Envoy does not qualify.
+1. Be either:
+   1. An actively maintained distribution of Envoy components. An example is
+   "SuperAwesomeLinuxDistro" which offers Envoy pre-built packages. Another example is
+   "SuperAwesomeServiceMesh" which offers a service mesh product that includes Envoy as a component.
+   2. OR offer Envoy as a publicly available infrastructure or platform service, in which the
+   product clearly states that it is built on top of Envoy. E.g., "SuperAwesomeCloudProvider's
+   Envoy as a Service (EaaS)". An infrastructure service that uses Envoy for a product but does not
+   publicly say they are using Envoy does not qualify. This is essentially IaaS or PaaS, if you use
+   Envoy to support a SaaS, e.g. "SuperAwesomeCatVideoService", this does not qualify.
 2. Have a user base not limited to your own organization.
```

Comment:
Per previous discussion, should we change this to a requirement on the minimum number of users?

The whole point behind distributors list is to protect the highest number of users while minimizing the risk of leaking any information about existing vulnerabilities, and adding "SuperAwesomeCloudProvider" that only has a handful of customers is probably not worth the risk.

Reply:
Yes, we should ask for customer size in the application. I don't want to put an exact bound on this in the policy, but we can consider this alongside other aspects of the application, e.g. how actively involved the folks are in the community and their history of handling security issues.